test/framework/batch_helper.hpp

```diff
@@ -195,6 +195,97 @@ namespace framework {
       return framework::expected::val(result_batch).value().value;
     }
 
+    /**
+     * Namespace provides useful functions which are related to implementation
+     * but they are internal API
+     */
+    namespace internal {
+
+      /**
+       * Create list of hashes
+       * @tparam TxBuilderCollection - type of builder
+       * @param builders - initializer list which contains all builders
+       * @return vector with transactions hashes
+       */
+      template <typename TxBuilderCollection>
+      auto fetchReducedHashes(
+          const std::initializer_list<TxBuilderCollection> &builders) {
+        std::vector<shared_model::interface::types::HashType> hashes;
+        std::for_each(
+            builders.begin(), builders.end(), [&hashes](const auto &builder) {
+              hashes.push_back(builder.build().reducedHash());
+            });
+        return hashes;
+      }
+
+      /**
+       * Variadic to initializer list adapter
+       */
+      template <typename... TxBuilders>
+      auto fetchReducedHashes(const TxBuilders &... builders) {
```

Comment:
Seems that `TxBuilders` and `TxBuilderCollection` represent the same type. For the purposes of clarity and consistency, please use the same name in both places.